lib/common.mli

```diff
@@ -1,8 +1,10 @@
 val version : string
 
 exception Json_error of string
+(** Exception (usually) describing a parsing failure. *)
```

Comment:
I think the "usually" should be qualified a bit better because I think now it is adding more questions than answering those.

Reply:
I just expanded the description to list both usages.